cart/data-cart.js

```diff
@@ -0,0 +1,39 @@
+import { items, cart } from '../data.js';
+import { findById } from '../utils.js';
+
+const total = calculateTotal(cart);
+
+//const totalCell = document.querySelector('.total');
+document.getElementById('total').innerHTML = total;
+//totalCell.textContent = `Total: $${total}`;
+
+function calculateTotal(cartArray) {
```

Comment:
wonderful job! Just looking for testing of these functions as well :)

Reply:
Ok thank you I actually have part of one wrote out just not finished yet 😪   and before lunch I got it all linked together... yay!!!!!!! Thank you so much